frontend/public/index.html

```diff
@@ -25,10 +25,6 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
 
-    <!-- Leaflet CSS -->
-    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
-      integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
-      crossorigin="" />
```

Comment:
Leaflet CSS has been moved to an import statement in `index.tsx` - we already have it locally in `node_modules`